### ISSUE TITLE
macos build fails on codesign

### ISSUE THREAD

**Issue by rgaudin** [kiwix/kiwix-hotspot#145]
In GitLab by @thiolliere on Dec 12, 2017, 13:02

I didn't investigate yet:
```
$ codesign -s "Developer ID Application: Wikimedia CH (L7HWM3SP3L)" dist/pibox-installer.app --deep

No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.

Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received

The build has been terminated
```

**Comment by rgaudin** [comment]
In GitLab by @thiolliere on Dec 22, 2017, 09:43

mentioned in merge request !27

**Comment by rgaudin** [comment]
In GitLab by @thiolliere on Dec 22, 2017, 09:43

closed via commit 7b1179e35c84349a0646e7e4f63d9e352472753a

**Comment by rgaudin** [comment]
In GitLab by @thiolliere on Dec 22, 2017, 09:43

closed via merge request !27
